oak/common/label.cc

```diff
@@ -59,6 +59,13 @@ oak::label::Label WebAssemblyModuleLabel(const std::string& web_assembly_module_
   return label;
 }
 
+oak::label::Label TlsEndpointLabel(const std::string& authority) {
```

Comment:
Does authority allow wildcards `*`?

Reply:
No, currently it's literal `host:port`.

Reply:
Does it propagate to subdomains?

Reply:
No, it is just `host:port` exactly as it appears in the URL.